static/index.html

```diff
@@ -247,8 +247,7 @@ <h2 class="response-title">Test Ether sent to the wallet</h2>
   <div class="provider-url">Node url: {{ public_node_url }}</div>
   <script>
     const faucetApp = (function () {
-      let providerUrl = "{{ public_node_url }}";
-      let blockExplorer = "https://fuellabs.github.io/block-explorer-v2";
+      let blockExplorer = "https://app.fuel.network/";
```

Comment:
I think you may need to remove this trailing slash, as a slash is added in the url template later on

Reply:
I tested it and it works fine, but I removed just in case